py-scripts/lf_base_interop_profile.py

```diff
@@ -1414,7 +1414,7 @@ async def configure_wifi(self, select_serials=None):
         return [self.selected_devices, self.report_labels, self.selected_macs]
 
     async def configure_wifi_groups(self, select_serials, serials_input, ssid_input, passwd_input, enc_input, eap_method_input,
-                                    eap_identity_input, ieee80211, key_management, private_key, ca_cert, client_cert, pk_passwd, pac_file):
+                                    eap_identity_input, ieee80211=True, key_management='DEFAULT', private_key='NA', ca_cert='NA', client_cert='NA', pk_passwd='NA', pac_file='NA'):
```

Comment:
Can you please resubmit with Verified CLI, as per patch requirements in Confluence? CLI from a script which includes and uses this would be sufficient.